data/db-schema/src/main/resources/net/corda/db/schema/config/migration/cpx-creation-v1.0.xml

```diff
@@ -143,15 +167,18 @@
             </column>
         </createTable>
 
-        <addPrimaryKey tableName="cpk_db_change_log" columnNames="cpk_file_checksum, file_path" constraintName="cpk_db_change_log_pk"
-                       schemaName="${schema.name}"/>
+        <addPrimaryKey tableName="cpk_db_change_log" columnNames="cpk_name, cpk_version, cpk_signer_summary_hash, file_path"
+                       constraintName="cpk_db_change_log_pk" schemaName="${schema.name}"/>
 
-        <addForeignKeyConstraint baseTableName="cpk_db_change_log" baseColumnNames="cpk_file_checksum"
-                                 referencedTableName="cpk_metadata" referencedColumnNames="file_checksum"
+        <addForeignKeyConstraint baseTableName="cpk_db_change_log" baseColumnNames="cpk_name, cpk_version, cpk_signer_summary_hash"
+                                 referencedTableName="cpk_metadata" referencedColumnNames="cpk_name, cpk_version, cpk_signer_summary_hash"
                                  constraintName="FK_cpk_db_change_log_cpk_metadata"
                                  baseTableSchemaName="${schema.name}"
                                  referencedTableSchemaName="${schema.name}"/>
 
+        <addUniqueConstraint tableName="cpk_db_change_log" columnNames="cpk_file_checksum" constraintName="db_cpk_db_change_log_uc1"
+                             schemaName="${schema.name}"/>
+
         <createTable tableName="cpi_cpk" schemaName="${schema.name}">
```

Comment:
```
create table cpi_cpk
(
    cpi_name                varchar(255)            not null,
    cpi_version             varchar(255)            not null,
    cpi_signer_summary_hash varchar(255)            not null,
    cpk_name                varchar(255)            not null,
    cpk_version             varchar(255)            not null,
    cpk_signer_summary_hash varchar(255)            not null,
    cpk_file_checksum       varchar(255)            not null
        constraint db_cpi_cpk_uc1
            unique,
    cpk_file_name           varchar(255)            not null,
    entity_version          integer                 not null,
    is_deleted              boolean                 not null,
    insert_ts               timestamp default now() not null,
    constraint cpi_cpk_pk
        primary key (cpi_name, cpi_version, cpi_signer_summary_hash, cpk_file_checksum),
    constraint "FK_cpi_cpk_cpi"
        foreign key (cpi_name, cpi_version, cpi_signer_summary_hash) references cpi,
    constraint "FK_cpi_cpk_cpk_metadata"
        foreign key (cpk_name, cpk_version, cpk_signer_summary_hash) references cpk_metadata
);
```

Reply:
If `cpk_signer_summary_hash` is nullable as the PK for CPK tables, will have to make this nullable too